unblob/handlers/compression/lz4.py

```diff
@@ -89,7 +91,11 @@ def calculate_chunk(self, file: File, start_offset: int) -> Optional[ValidChunk]
                 file.seek(-4, io.SEEK_CUR)
                 break
 
-            file.seek(block_compressed_size, io.SEEK_CUR)
+            compressed_block = file.read(block_compressed_size)
+            uncompressed_block = decompress(compressed_block, MAX_LEGACY_BLOCK_SIZE)
+
+            if len(uncompressed_block) < MAX_LEGACY_BLOCK_SIZE:
+                break
```

Comment:
I would just add a comment with a reference on this change

Reply:
Done :)